pkg/releasetesting/environment.go

```diff
@@ -19,17 +19,21 @@ package releasetesting
 import (
 	"bytes"
 	"fmt"
+	"io/ioutil"
 	"log"
+	"strings"
 	"sync"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 
 	"k8s.io/helm/pkg/proto/hapi/release"
 	"k8s.io/helm/pkg/proto/hapi/services"
 	"k8s.io/helm/pkg/tiller/environment"
 )
 
+var logEscaper = strings.NewReplacer("%", "%%")
```

Comment:
@bacongobbler everything lgtm - can you just explain/add a comment for this?

Reply:
For sure!